jobs/sql/extractions/appointments.sql

```diff
@@ -0,0 +1,90 @@
+set @partition = '${partitionNum}';
+
+DROP temporary TABLE IF EXISTS temp_appointments;
+create temporary table temp_appointments
+(
+    appointment_id       int,
+    patient_id           int,
+    emr_id               varchar(50),
+    date_issued          date,
+    location             varchar(255),
+    service              varchar(50),
+    type                 varchar(45),
+    recurring            bit,
+    appointment_datetime datetime,
+    duration             int,
+    provider             varchar(255),
+    status               varchar(45),
+    note                 varchar(255),
+    datetime_created     datetime,
+    user_entered         varchar(255),
+    index_asc            int,
+    index_desc           int
+);
+
+insert into temp_appointments(
+      appointment_id,
+      patient_id,
+      date_issued,
+      location,
+      service,
+      type,
```

Comment:
Right now the only "type" we are supporting is "Scheduled".  (The three types available are "Scheduled", "Walkin" and "Virtual").  As this field has little meaning in our implememtation, I'm wondering if we want to skip including it to avoid people getting confused between a "type" of "Scheduled" and a "status" of "Scheduled".